app/init/init_gatekeeper.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from core import settings
+
+def register_apis_to_gatekeeper():
+
+    # Login
+    at = requests.post(
+        url=settings.GATEKEEPER_BASE_URL.unicode_string() + "api/login/",
+        headers={"Content-Type": "application/json"},
+        json={"username": "admin", "password": "admin"}
```

Comment:
this must not be hardcoded

Reply:
Yeah, forgot to read from .env, I'll update it with proper information